src/ctlab2.js

```diff
@@ -0,0 +1,65 @@
+/*
+export function map(arr, callback) {
+  const newArr = [];
+
+  for(let i = 0; i < arr.length; i++) {
+    const newElem = callback(elem);
```

Comment:
Nice work could clean up the hidden code and i am really enjoying the notes and steps on how you get there amazing!